zaza/openstack/charm_tests/ceph/fs/tests.py

```diff
@@ -103,6 +106,73 @@ def verify(stdin, stdout, stderr):
             'sudo cat /mnt/cephfs/test',
             password=password, privkey=privkey, verify=verify)
 
+    def test_conf(self):
+        """Test ceph to ensure juju config options are properly set."""
+        self.TESTED_UNIT = 'ceph-fs/0'
+
+        def _get_conf():
+            """get/parse ceph daemon response into dict for specified configs.
+
+            :returns dict: conf options selected from configs
+            :rtype: dict
+            """
+            configs = ["mds_cache_memory_limit",
+                       "mds_cache_reservation",
+                       "mds_health_cache_threshold"]
+            holder = {}
+            for config in configs:
+                cmd = "sudo ceph daemon mds." \
+                      "$HOSTNAME config show | grep {}".format(config)
+                conf = model.run_on_unit(self.TESTED_UNIT, cmd)
+                print(conf)
```

Comment:
It's generally preferred to use a log statement, rather than `print`

Reply:
Good catch -- I just removed it, not really needed outside debugging.